python/cudf/cudf/core/df_protocol.py

```diff
@@ -660,13 +660,8 @@ def from_dataframe(
     if not hasattr(df, "__dataframe__"):
         raise ValueError("`df` does not support __dataframe__")
 
-    return _from_dataframe(df.__dataframe__(allow_copy=allow_copy))
+    df = df.__dataframe__(allow_copy=allow_copy)
```

Comment:
Although `from_dataframe` previously didn't document the `allow_copy` argument, maybe now is a good time to do so?

Reply:
+1, please add parameter/return docstrings to `from_dataframe`.

Reply:
Done